src/NuGet.Core/NuGet.Configuration/PublicAPI.Shipped.txt

```diff
@@ -467,6 +467,7 @@ static readonly NuGet.Configuration.ConfigurationConstants.UserKey -> string
 static readonly NuGet.Configuration.ConfigurationConstants.UsernameToken -> string
 static readonly NuGet.Configuration.ConfigurationConstants.ValidAuthenticationTypesToken -> string
 static readonly NuGet.Configuration.ConfigurationConstants.ValueAttribute -> string
+static readonly NuGet.Configuration.NuGetConstants.AddV3TrackFile -> string
```

Comment:
Should it go first to PublicAPI.Unshipped.txt ?

Reply:
This came from the revert of the previous PR.  In other words, we used to have this public API, and the previous PR removed it.  I feel like if we're going to bring it back, we should treat it like a shipped API, since our 5.11 packages, and earlier, had it.